k8s/predict/train-job/values-stage.yaml

```diff
@@ -4,6 +4,7 @@ app:
   configmap: env-train-job-staging
 
 schedule: 0 1 1 * *
+suspend: true
```

Comment:
Suspend training in staging. This is done manually as highlighted by @sserurich

Reply:
Thanks @123MwanjeMike , does this mean that the manual process with partly continue...in staging? Please shed some more light on this. Thanks.

Reply:
Yes @Baalmart 
The automated training of the model is not a requirement for the staging environment as highlighted by @sserurich